bench_test/CMakeLists.txt

```diff
@@ -31,7 +37,7 @@ macro(define_runtime_algebra_test name)
     cm_test(NAME ${test_name} SOURCES ${name}.cpp ARGS ${additional_args})
 
     target_include_directories(${test_name} PRIVATE
-                               "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+                               "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>"
```

Comment:
It's a bed pattert to use any files not from the current dir or its subdirs in CMake. Move those includes into a different interface target

Reply:
fixed in 087efb2c2cdb89e3525bd999b7ceba2e9f9a85ce